docs/api.rst

```diff
@@ -818,6 +818,25 @@ Integrations
     :param payload: The raw event payload data.
     :type payload: :class:`RawIntegrationDeleteEvent`
 
+Audit Logs
+++++++++++
+
+.. function:: on_audit_log_entry_create(entry)
+
+    Called when an audit log entry is created.
+    You must have the :attr:`~Permissions.view_audit_log` permission to receive this.
+
+    This requires :attr:`Intents.moderation` to be enabled.
+
+    .. warning::
+        Because of a Discord limitation, :attr:`AuditLogEntry.target` will frequently
+        be of type :class:`Object` instead of the respective model.
```

Comment:
```suggestion
    .. warning::
        This scope of data in this gateway event is limited, which means it is much more
        reliant on the cache than :meth:`Guild.audit_logs`.
        Because of this, :attr:`AuditLogEntry.target` will frequently
        be of type :class:`Object` instead of the respective model.
```

Reply:
thanks a lot